src/components/Form/Form.jsx

```diff
@@ -185,7 +185,7 @@ const Form = ({recaptchaRef}) => {
           className={`${styles.button}`}
           onClick={(e) => clickHandler(e)}
         >
-          { disable || !token || token === '' ? 'Complete Form' : 'Send Message' }
+          { disable || !token || token === '' ? 'Complete Form'.toUpperCase() : 'Send Message' }
```

Comment:
While that is a JS method, I'm not sure that'll work here.  It may be easier to just type in the capital letters.

Reply:
Pls make same change to both button text options.  And awesome job linking the issue with the ticket!

Reply:
The method worked when I checked the changes. I can just type it out since it's just for this one button.

Also, are you considering making all buttons uppercase? If so, is this something we could consider creating a component for reuse? Or would you just add the text-transform property to all the button styles in the css files?

Reply:
Ok, gotcha.  Sorry, didn't actually pull and check it. 

We could, but there's only two buttons on the whole site and they each have unique functionality. This one is tied into the form logic and api call.

Either way, the text would be unique, so even if it was a component the text would be passed as a property of that component and would need to be typed out or use a method.

If we use a method, I believe the client browser would need to run that logic.  IMHO it's less for the client browser, as opposed to using methods for all text changes like this.

Reply:
Ahh, thanks for clarifying. I agree. Pushed the new changes, ready for review now :)